README.md

```diff
@@ -61,6 +63,91 @@ make run
 
 **NOTE:** You can also run this in one step by running: `make install run`
 
+#### Reconcile OLSConfig Custom Resource
+
+The namespace `openshift-lightspeed` should be created before this test.
+
+1. Create secret containing the API Key for BAM or OpenAI. The key for OpenAI is `openai_api_key.txt`, while
+   the key for BAM is `bam_api_key.txt`.
+
+```yaml
+apiVersion: v1
+data:
+  openai_api_key.txt: <base64 encoded API Key>
```

Comment:
We should have a generic format for the secrets users will have to create, regardless of whether they are using BAM, Watson, or OpenAI, etc.

it's possible at some point we'll have a provider that requires a different format(or multiple keys like "username" in addition to the token), but for now all the providers just require a token, so we should expect that users:

1) create a secret with a key named `apitoken` or some other generic key name
2) reference the appropriate secret from the provider config section

that way people don't have to look up how to create a secret based on which provider the secret is for.

Reply:
will use `apitoken` as the key for api token file for all providers.